doc/source/code-examples/advancedexamples.rst

```diff
@@ -1261,9 +1261,9 @@ As observable we can simply take :math:`Z_0 Z_1 Z_2` :
 
    from qibo.symbols import Z
    from qibo.hamiltonians import SymbolicHamiltonian
-   from qibo.backends import GlobalBackend
+   from qibo.backends import _Global
 
-   backend = GlobalBackend()
+   backend = _Global.backend()
```

Comment:
However, the current `get_backend()` returns `str(_Global.get_backend())`. I believe it would be better to branch based on a flag.

```py
def get_backend(name=False):
    if name:
        return str(_Global.backend())
    return _Global.backend()
```

Reply:
Updated this.